src/types.ts

```diff
@@ -16,9 +16,19 @@ export type RenderEnzyme<
   ...testProps: ConditionallyRequiredTestProps<Component, Props>
 ) => RenderEnzymeReturn<Component>;
 
+// Just like RenderEnzyme, but RenderRtl also allows callers to do `renderRtl.options({...})(props)`
+// In order to support both signature types, we extend the function's base type and add the options
+// attribute for those "in the know" ;)
 export type RenderRtl<
   Component extends React.ComponentType,
   Props extends Partial<FullProps<Component>>
+> = RenderRtlMethod<Component, Props> & {
+  options: (options: RenderOptions) => RenderRtlMethod<Component, Props>;
+};
```

Comment:
Ah, by not putting `options` on the `RenderRtlMethod` itself, folks won't be able to do this:

```ts
renderView.options(first).options(second)
```

...which I would guess to be a confusing hinderance.

Reply:
Oh, is that something we're intending? Based on the implementation, if they did that, `first` would be completely overwritten, so I didn't think that was a thing we wanted to make available?

Reply:
I was hoping to let people experiment and see what happens... 😈 

It's part of the JS side behavior of it.

Reply:
![image](https://user-images.githubusercontent.com/13091526/106926904-95afd200-66df-11eb-96c6-ceb9beb8d77c.png)